unified-codes/apps/data-service/src/app/v2/types/property/DrugInteractionProperty.ts

```diff
@@ -0,0 +1,45 @@
+import IProperty, { Property, EPropertyType } from './Property';
```

Comment:
I don't think the `Property` in the name adds anything; `DrugInteraction` would be enough for me.

Reply:
It's just to start a convention, just like entities are all appended with `Entity` (just makes it clear that they inherit from the respective base class). Does that justify?

Reply:
You're so conventional. let's run with it.